### PR TITLE
AnyPhone for Web (Express)

### PR DESCRIPTION
**Pull request by ericnakagawa**
AnyPhone can now be used over current desktop and mobile browsers!

### CODE REVIEW

File: cloud/app.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+app.get('/', function(req, res) {
+  res.render("index.ejs", {})
+});
+
+app.post('/', function(req, res) {
+  var form = req.body;
+  var phoneNumber = form.phoneNumber;
+
+  if(phoneNumber) {
```
Comment (gfosco) [inline cloud/app.js]: Please check that it is 10 digits and warn if not.

File: cloud/app.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+app.get('/verify', function(req, res){
+  res.redirect("/");
+})
+
+app.post('/verify', function(req, res){
+  var form = req.body;
+  var code = form.code;
+  var phoneNumber = form.phoneNumber;
+
+  if(code) {
```
Comment (gfosco) [inline cloud/app.js]: Check that this is 4 digits.  These validations could be done before submission too.

File: cloud/app.js (hunk 1 above)
Comment (ericnakagawa) [inline cloud/app.js]: Resolved in new commit.

File: cloud/app.js (hunk 2 above)
Comment (ericnakagawa) [inline cloud/app.js]: Resolved in newest commit.

**Pull request merged**